README.md

```diff
@@ -21,3 +21,13 @@ Crowbar documentation is distributed into multiple places under the /doc directo
 
 > Please, do NOT add documentation in locations besides /doc!  If necessary, expand this README to include pointers to important /doc information.
 
+## Development Environment
```

Comment:
That should be placed somewhere else in ```doc``` folder with a bigger overview how to develop on crowbar.